check_process

```diff
@@ -9,25 +9,23 @@
         setup_root=0
         setup_nourl=0
         setup_private=0
-        setup_public=0
+        setup_public=1
         upgrade=1
+		upgrade=1	from_commit=bdac42b709cd2bc663a68422f69ac6bfd43a1ce5
         backup_restore=1
         multi_instance=0
```

Comment:
Consider testing that feature, your app allows multi instance https://github.com/yalh76/z-push_ynh/blob/testing/manifest.json#L22

Reply:
I forgot to activate multi_instance check, I will and test it and update the test